react-app/src/Pages/Register Page/RegisterPage.js

```diff
@@ -0,0 +1,58 @@
+import './RegisterPage.css';
+import NavBar from '../../Components/navBar/navBar';
+import { TextField, Typography, Button } from '@mui/material';
+import { useState } from 'react';
+import axios from 'axios';
+
+function LoginPage() {
+    const [username, setUsername] = useState('')
+    const [password, setPassword] = useState('')
+
+    const submitDetails = (e) => {
+        e.preventDefault()
+
+        if (username && password) {
+            console.log(username, password)
+        }
+        else {
+            console.log('error')
+        }
+    }
+
+    const registerDetails = async(e) => {
+        console.log('regiseter=')
+        await axios.post('http://localhost:3001/auth/register', {username, password});
+    }
+
+    return (
+        <div>
+            <NavBar />
+            <Typography variant="h3">Register</Typography>
+            <div class='log-in-box'>
+                <form noValidate autoComplete='off' onSubmit={submitDetails}>
+                    <Typography variant="body1">Register</Typography>
+                    <Typography variant="body1">Username</Typography>
+                    <TextField 
+                        id="standard-basic"
+                        onChange={(e) => setUsername(e.target.value)} 
+                        variant="standard" 
+                        required
+                    />
+                    <br />
+                    <br />
+                    <Typography variant="body1">Password</Typography>
+                    <TextField 
+                        id="standard-basic"
+                        onChange={(e) => setPassword(e.target.value)}
+                        variant="standard" 
+                        required
+                    />
+                    <Button variant="contained" onClick={login}>Log In</Button>
```

Comment:
Does this on click actually do anything? Login function not defined above that I can see.

Reply:
I was going to separate the login and register pages.